nodes/src/nodes/memory_persistent/memory_store.py

```
@@ -0,0 +1,710 @@
+
+    def _history_key(self, session_id: str) -> str:
+        return f'{_REDIS_PREFIX}:{session_id}:__history__'
+
+    def _meta_key(self, session_id: str) -> str:
+        return f'{_REDIS_PREFIX}:{session_id}:__meta__'
+
+    def _set_ttl(self, session_id: str, ttl_seconds: Optional[float]) -> None:
+        """Apply TTL to all keys belonging to a session."""
+        if ttl_seconds is None or ttl_seconds <= 0:
+            return
+        ttl_ms = max(1, int(round(ttl_seconds * 1000)))
```

Ruff (0.15.9)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)